code/data.py

```diff
@@ -0,0 +1,80 @@
+'''
+Provide processed data ready for consumption by tensorflow model
+Return data always as numpy array
+'''
+
+import numpy as np
+import os
+from collections import Counter
+
+class NlpData:
+
+    data = None
+    data_only_ids = None
+    vocab = None
+    current_batch = 0
+
+    def get_next_batch(self, batch_size):
```

Comment:
I think an iterator would be nice here. Check out what I did here: https://github.com/muyajil/nlp-project-17/blob/gideon/preprocess.py#L71.